### DISCUSSION TITLE
Can we use github with just a few whitelisted Repos?

### DISCUSSION THREAD

**Discussion by EpicPhelan** [community/community#109224] [GitHub Education]
### Hi everyone,

I've been using GitHub with my advanced programmers, but now the tech department has blocked it. They said if I could get it to work with just the repos I use with my class, they would white list those URLS. I can't figure it out. It seems we will still have to authenticate with Github and since those servers are blocked, we will not be allowed to do anything to the repos. Am I missing something? 

They are mad about students using code they found on Github to get around our school's filtering software.

**Comment by Vijaykv5** [comment]
Hi @EpicPhelan 

Yes, it's possible to use GitHub with only a few whitelisted repositories. The tech department can whitelist specific repository URLs, allowing access to those repositories while keeping other GitHub content blocked. However, accessing these repositories might still require authentication, which could be problematic if GitHub's servers are blocked. 

To address this, setting up a private GitHub organization exclusively for the class's repositories could streamline access. Clear communication with the tech department to understand their concerns and ensure compliance with school policies is essential.
